src/System.Globalization/tests/PlatformDetection.cs

```diff
@@ -12,6 +12,8 @@ public static class PlatformDetection
     public static bool IsWindows => RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
     public static bool IsOSX => RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
     public static bool IsNetBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
+    public static bool IsLinux => RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
+    public static bool IsUnix => IsLinux || IsOSX;
```

Comment:
This would be better as IsUnix => !IsWindows.  We have people bringing up other platforms, like NetBSD, for which IsUnix should return true, and we don't want to have to add each of them to this ||'d list.